src/migration/1660550354504-TvlGroupEnum.ts

```diff
@@ -4,10 +4,10 @@ export class TvlGroupEnum1660550354504 implements MigrationInterface {
   name = 'TvlGroupEnum1660550354504'
 
   public async up (queryRunner: QueryRunner): Promise<void> {
-    await queryRunner.query('ALTER TYPE "public"."tvl_tvlgroup_enum" RENAME TO "tvl_tvlgroup_enum_old"')
-    await queryRunner.query('CREATE TYPE "public"."tvl_tvlgroup_enum" AS ENUM(\'tvlAmm\', \'tvlLending\', \'tvlProtocol\', \'tvlSubprotocols\', \'tvlStaking\', \'tvlZero\')')
-    await queryRunner.query('ALTER TABLE "tvl" ALTER COLUMN "tvlGroup" TYPE "public"."tvl_tvlgroup_enum" USING "tvlGroup"::"text"::"public"."tvl_tvlgroup_enum"')
-    await queryRunner.query('DROP TYPE "public"."tvl_tvlgroup_enum_old"')
+    // await queryRunner.query('ALTER TYPE "public"."tvl_tvlgroup_enum" RENAME TO "tvl_tvlgroup_enum_old"')
+    // await queryRunner.query('CREATE TYPE "public"."tvl_tvlgroup_enum" AS ENUM(\'tvlAmm\', \'tvlLending\', \'tvlProtocol\', \'tvlSubprotocols\', \'tvlStaking\', \'tvlZero\')')
+    // await queryRunner.query('ALTER TABLE "tvl" ALTER COLUMN "tvlGroup" TYPE "public"."tvl_tvlgroup_enum" USING "tvlGroup"::"text"::"public"."tvl_tvlgroup_enum"')
+    // await queryRunner.query('DROP TYPE "public"."tvl_tvlgroup_enum_old"')
```

Comment:
We can remove this code if it's not going to be used in future, unless relevant for future reference (in which case you can close this comment)

Reply:
Yep, should be uncommented